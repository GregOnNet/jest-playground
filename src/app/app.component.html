<h1>{{ title }}!</h1>
<p>
  Welcome to the playground, you will find all testing samples in
  <code>*.spec.ts</code> files.
</p>
