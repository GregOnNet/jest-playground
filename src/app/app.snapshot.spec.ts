import { ComponentFixture, TestBed } from '@angular/core/testing';
import { AppComponent } from './app.component';

describe('<app-root>', () => {
  let sut: ComponentFixture<AppComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [AppComponent]
    });

    sut = TestBed.createComponent(AppComponent);
    sut.detectChanges();
  });

  describe('When the application stats', () => {
    it('displays the welcome message', () => {
      expect(sut).toMatchSnapshot();
    });
  });
});
