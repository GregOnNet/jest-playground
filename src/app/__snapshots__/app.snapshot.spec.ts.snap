// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<app-root> When the application stats displays the welcome message 1`] = `
<app-root
  title={[Function String]}
>
  <h1>
    Jest Playground!
  </h1><p>
     Welcome to the playground, you will find all testing samples in 
    <code>
      *.spec.ts
    </code>
     files.
  
  </p>
</app-root>
`;
